src/Symfony/Component/Form/composer.json

```diff
@@ -33,7 +33,10 @@
         "symfony/http-foundation": ""
     },
     "autoload": {
-        "psr-0": { "Symfony\\Component\\Form\\": "" }
+        "psr-0": { "Symfony\\Component\\Form\\": "" },
+        "exclude-from-classmap": [
+            "/Tests/"
```

Comment:
Shouldn't `Test`be included in `autoload-dev` only? As you said there are only meant to be used for tests, so you don't need them in production mode.